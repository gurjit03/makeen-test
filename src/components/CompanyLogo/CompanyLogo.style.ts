import styled from 'styled-components'
import { colors, fontSizes } from '../../config/jss'

export const Logo = styled.h3`
	font-size: ${fontSizes.large}px;
	text-transform: uppercase;
	color: ${colors.white};
`
