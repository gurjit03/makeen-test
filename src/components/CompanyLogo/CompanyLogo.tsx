import React from 'react'
import { Logo } from './CompanyLogo.style'

const CompanyLogo: React.FC = () => <Logo>COMATCH</Logo>

export default CompanyLogo
