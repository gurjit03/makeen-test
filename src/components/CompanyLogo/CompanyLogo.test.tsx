import React from 'react';
import { render } from '@testing-library/react'
import CompanyLogo from './CompanyLogo';

describe('<CompanyLogo />', () => {
    it('should render properly', () => {
        const { asFragment } = render(<CompanyLogo />)
        expect(asFragment()).toMatchSnapshot();
    });
})