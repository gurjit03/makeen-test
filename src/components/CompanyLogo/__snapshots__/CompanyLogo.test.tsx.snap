// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompanyLogo /> should render properly 1`] = `
<DocumentFragment>
  <h3
    class="sc-bdfBQB epGijZ"
  >
    COMATCH
  </h3>
</DocumentFragment>
`;
